test/performance/latency-controller/README.md

```diff
@@ -1,242 +1,177 @@
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 80
-          initialDelaySeconds: 5
-          periodSeconds: 5
-EOF
-```
+## Features
 
-```bash
-# Deployment 2 (variation)
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: nginx-test-deployment
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: nginx
-  template:
-    metadata:
-      labels:
-        app: nginx
-    spec:
-      containers:
-      - name: nginx
-        image: nginx:alpine
-        ports:
-        - containerPort: 80
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 80
-          initialDelaySeconds: 5
-          periodSeconds: 5
-EOF
-```
+### Comprehensive Latency Metrics
 
-### Pods
+The controller tracks multiple stages of the deployment pipeline:
 
-Apply pods to generate observable latency metrics:
+- **📦 Packaging Duration**: Time from WDS object creation to ManifestWork creation  
+- **📬 Delivery Duration**: Time from ManifestWork to AppliedManifestWork creation  
+- **🏭 Activation Duration**: Time from AppliedManifestWork to WEC object creation  
+- **⚡ Total Downsync Duration**: End-to-end time from WDS to WEC object creation  
+- **📝 Status Propagation**: Time for status to flow back from WEC to WDS  
+- **🔄 End-to-End Latency**: Complete cycle from workload creation to status update  
 
-```bash
-# Pod 1
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: v1
-kind: Pod
-metadata:
-  name: nginx-pod
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  containers:
-  - name: nginx
-    image: nginx:alpine
-    ports:
-    - containerPort: 80
-    readinessProbe:
-      httpGet:
-        path: /
-        port: 80
-      initialDelaySeconds: 5
-      periodSeconds: 5
-EOF
-```
+### Advanced Monitoring Capabilities
 
-```bash
-# Pod 2 (variation)
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: v1
-kind: Pod
-metadata:
-  name: nginx-pod-2
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  containers:
-  - name: nginx
-    image: nginx:alpine
-    ports:
-    - containerPort: 80
-    readinessProbe:
-      httpGet:
-        path: /
-        port: 80
-      initialDelaySeconds: 5
-      periodSeconds: 5
-EOF
-```
+- **Multi-Cluster Support**: Monitors workloads across multiple WEC clusters simultaneously  
+- **Dynamic Resource Discovery**: Automatically discovers and monitors all configured Kubernetes resource types  
+- **Real-time Metrics**: Provides live latency measurements via Prometheus metrics  
+- **Workload Count Tracking**: Monitors the number of deployed workload objects per cluster  
+- **Status Field Detection**: Intelligently handles resources with and without status fields  
 
-## 🛠️ Build & Run
+## Prerequisites
```

Comment:
Let's move the Pre-requisites section at the begining of the README. Just below the figure and before the "Overview" section.